### PR TITLE
Focal point config exports

### LINKED ISSUE #1338: Update focal_point config
We are using the normal styles effects but in order to use focal point we need to migrate them to the new styles:

focal point crop
focal point scale and crop

### PR DESCRIPTION
**Pull request by AndyEPhipps**
Fixes #1338 

Reasserts the correct Focal Point widget for the Background Single Msg row and Landing pages

### CODE REVIEW

**Comment by AndyEPhipps** [comment]
Once approved, this can be updated with the proper Campaign version (rather than this PR commit): https://github.com/comicrelief/sr18/pull/265

File: modules/cr_rich_text_paragraph/config/install/core.entity_form_display.paragraph.cr_rich_text_paragraph.default.yml [hunk 1]
```diff
@@ -6,10 +6,10 @@ dependencies:
     - field.field.paragraph.cr_rich_text_paragraph.field_bg_single_msg_bg_colour
     - field.field.paragraph.cr_rich_text_paragraph.field_bg_single_msg_layout
     - field.field.paragraph.cr_rich_text_paragraph.field_body
-    - image.style.large
+    - image.style.medium
```
Comment (Saphyel) [inline modules/cr_rich_text_paragraph/config/install/core.entity_form_display.paragraph.cr_rich_text_paragraph.default.yml]: is this change expected?

Comment (Heleen-cr) [reply modules/cr_rich_text_paragraph/config/install/core.entity_form_display.paragraph.cr_rich_text_paragraph.default.yml]: I thought this was deliberate. It's the image style used for the preview image, so makes sense to use a smaller image style for that.

Comment (AndyEPhipps) [reply modules/cr_rich_text_paragraph/config/install/core.entity_form_display.paragraph.cr_rich_text_paragraph.default.yml]: As all of this config had been reverted/broken, I just used RND as reference point, and that uses the Medium preview size

**Pull request merged**